### PR TITLE
An example web app.

### PR DESCRIPTION
**Pull request by OleksiiZubko**
An example web app shows how to use Here Data SDK for TypeScript
to upload and publish large data in a browser.

Resolves: OLPEDGE-2501
Relates-To: OLPEDGE-2456

Signed-off-by: Oleksii Zubko <ext-oleksii.zubko@here.com>

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ HERE Data SDK for TypeScript contains several examples that demonstrate some of
 - [Authorization example](authorization-example/README.md) – build the Data SDK using webpack and work with the APIs from `@here/olp-sdk-dataservice-api`. You can use this example application to find the list of groups that you have access to as well as create groups.
 - [React App example](react-app-example/README.md) – use the Data SDK and React in a browser and learn how to work with the following modules: `olp-sdk-authentication`, `olp-sdk-dataservice-read`, and `olp-sdk-dataservice-write`.
 - [Node.js example](nodejs-example/README.md) – work with the Data SDK in Node.js and learn how to get data from versioned layers and save it to files.
+- [MultiPartUploadWrapper example](multipart-upload-wrapper-example/README.md) – an example web app shows how to use Here Data SDK for TypeScript to upload and publish large data in a browser.
```
Comment (andescu) [inline examples/README.md:8]:
```suggestion
- [MultiPartUploadWrapper example](multipart-upload-wrapper-example/README.md) – an example web app that shows how to use Here Data SDK for TypeScript to upload and publish large data in a browser.
```

File: examples/multipart-upload-wrapper-example/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# An example web app shows how to use Here Data SDK for TypeScript to upload and publish large data in a browser.
```
Comment (andescu) [inline examples/multipart-upload-wrapper-example/README.md:1]:
```suggestion
# Publish large data in a browser.
```

File: examples/multipart-upload-wrapper-example/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+This is an HTML+JS UI and HTTP server with mocked routes to simulate the real DataStore.
```
Comment (andescu) [inline examples/multipart-upload-wrapper-example/README.md:3]: What are "mocked routes"?

File: examples/multipart-upload-wrapper-example/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (andescu) [inline examples/multipart-upload-wrapper-example/README.md:4]: Please use the same structure as for the other examples. Add two sections: **Setup** and **Run the app**

File: examples/multipart-upload-wrapper-example/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+UI uses the `HERE Data SDK for Typescript` in plain JS and you can see how to create `UserAuth`, `OlpClientSettings`, and `MultipartUploadWrapper` instances and use it to upload and publish data to the DataStore. You can play with more than 8GB of files because the browser does not load them to the memory.
```
Comment (andescu) [inline examples/multipart-upload-wrapper-example/README.md:5]:
```suggestion
The UI uses the `HERE Data SDK for Typescript` in plain JS. This example illustrates how to use the `MultipartUploadWrapper` class to upload and publish large files to a versioned layer. The `MultipartUploadWrapper` enables you to upload large files (more then 8GB) without loading them in memory upfront.
```

File: examples/multipart-upload-wrapper-example/README.md (hunk 3 above)
Comment (OleksiiZubko) [reply examples/multipart-upload-wrapper-example/README.md:3]: Mocked Blob and Publish endpoints and responses to simulate the real OLP

Comment (OleksiiZubko) [reply examples/multipart-upload-wrapper-example/README.md:3]: lines 47 - 118 https://github.com/heremaps/here-data-sdk-typescript/pull/460/files#diff-52da086ca44fc827896c5ba8c3e6d147db75e34b8abfa7dfa0c090316f9544c7R47

File: examples/multipart-upload-wrapper-example/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+## Run
+
+1. Start the server.
+
+```
+node server.js
+```
+
+2. Open in your favorite browser http://localhost:8080
```
Comment (HalynaDumych) [inline examples/multipart-upload-wrapper-example/README.md:23]:
```suggestion
# Publish large data files in a browser

This example app uses an HTML and JS UI and an HTTP server with mocked routes to simulate the real `DataStore` class.

You can use this app to learn how to work with the `MultipartUploadWrapper` class and publish large files of more than 8 GB to a versioned layer without loading them in memory.

## Setup

To set up the example app, go to the `multipart-upload-wrapper-example` folder .

## Run

1. Start the server.
    ```
    node server.js
    ```
2. In your favorite browser, open `http://localhost:8080/`.

```

**Review by HalynaDumych — CHANGES_REQUESTED** [review]

File: examples/README.md [hunk 7]
```diff
@@ -5,3 +5,4 @@ HERE Data SDK for TypeScript contains several examples that demonstrate some of
 - [Authorization example](authorization-example/README.md) – build the Data SDK using webpack and work with the APIs from `@here/olp-sdk-dataservice-api`. You can use this example application to find the list of groups that you have access to as well as create groups.
 - [React App example](react-app-example/README.md) – use the Data SDK and React in a browser and learn how to work with the following modules: `olp-sdk-authentication`, `olp-sdk-dataservice-read`, and `olp-sdk-dataservice-write`.
 - [Node.js example](nodejs-example/README.md) – work with the Data SDK in Node.js and learn how to get data from versioned layers and save it to files.
+- [MultiPartUploadWrapper example](multipart-upload-wrapper-example/README.md) – an example web app that shows how to use Here Data SDK for TypeScript to upload and publish large data in a browser.
```
Comment (HalynaDumych) [inline examples/README.md:8]:
```suggestion
- [MultiPartUploadWrapper example](multipart-upload-wrapper-example/README.md) – upload and publish large data in a browser.
```

File: examples/multipart-upload-wrapper-example/README.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# Publish large data files in a browser
+
+This example app uses an HTML and JS UI and an HTTP server with mocked routes to simulate the real `DataStore` class.
+You can use this app to learn how to work with the `MultipartUploadWrapper` class and publish large files of more than 8 GB to a versioned layer without loading them in memory.
+
+## Setup
+
+To set up the example app, go to the `multipart-upload-wrapper-example` folder .
```
Comment (andescu) [inline examples/multipart-upload-wrapper-example/README.md:8]:
```suggestion
This example does not need any setup. Nevertheless, you should have node.js installed.
```

File: examples/multipart-upload-wrapper-example/README.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+# Publish large data files in a browser
+
+This example app uses an HTML and JS UI and an HTTP server with mocked routes to simulate the real `DataStore` class.
+You can use this app to learn how to work with the `MultipartUploadWrapper` class and publish large files of more than 8 GB to a versioned layer without loading them in memory.
+
+## Setup
+
+This example does not need any setup. Nevertheless, you should have node.js installed.
```
Comment (HalynaDumych) [inline examples/multipart-upload-wrapper-example/README.md:8]:
```suggestion
This example does not need any setup. Nevertheless, you should have Node.js installed.
```

File: examples/README.md [hunk 10]
```diff
@@ -5,3 +5,4 @@ HERE Data SDK for TypeScript contains several examples that demonstrate some of
 - [Authorization example](authorization-example/README.md) – build the Data SDK using webpack and work with the APIs from `@here/olp-sdk-dataservice-api`. You can use this example application to find the list of groups that you have access to as well as create groups.
 - [React App example](react-app-example/README.md) – use the Data SDK and React in a browser and learn how to work with the following modules: `olp-sdk-authentication`, `olp-sdk-dataservice-read`, and `olp-sdk-dataservice-write`.
 - [Node.js example](nodejs-example/README.md) – work with the Data SDK in Node.js and learn how to get data from versioned layers and save it to files.
+- [MultiPartUploadWrapper example](multipart-upload-wrapper-example/README.md) – use the Data SDK to upload and publish large amount of data in a browser.
```
Comment (HalynaDumych) [inline examples/README.md:8]:
```suggestion
- [MultiPartUploadWrapper example](multipart-upload-wrapper-example/README.md) – use the Data SDK to upload and publish a large amount of data in a browser.
```

File: examples/multipart-upload-wrapper-example/README.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
+# Publish large data files in a browser
+
+This example app uses an HTML and JS UI and an HTTP server with mocked routes to simulate the real `DataStore` class.
+You can use this app to learn how to work with the `MultipartUploadWrapper` class and publish large files of more than 8 GB to a versioned layer without loading them in memory.
+
+## Setup
+
+This example does not need any setup. Nevertheless, you should have Node.js installed.
```
Comment (HalynaDumych) [inline examples/multipart-upload-wrapper-example/README.md:8]:
```suggestion
This example does not need any setup. Nevertheless, make sure you installed Node.js.
```

**Review by HalynaDumych — APPROVED** [review]

**Review by andescu — APPROVED** [review]

**Pull request merged**